src/main/java/org/linksharing/server/http/ApiRestController.java

```diff
@@ -67,7 +67,14 @@ ResponseEntity<UserProfileDetails> updateProfileDetails(Principal user, @Request
 
     @GetMapping("/profile/picture")
     ResponseEntity<?> getProfilePicture(Principal user) throws FileNotFoundException {
-        File img = new File("../image.jpg");
+
+        String imageUrl = "../image.jpg";
```

Comment:
Why did you change this method?
Do you want to implement the next task in this PR?

Reply:
No, no, no.
Just a little messed in my local branches, and it was too late, I lost my focus et.
Dicard it please.